infra-templates/k8s/18/docker-compose.yml

```diff
@@ -17,7 +17,8 @@ kubelet:
         - --cluster-domain=cluster.local
         - --network-plugin=cni
         - --network-plugin-dir=/etc/cni/managed.d
-    image: rancher/k8s:v1.5.2-rancher1
+        - --pod-infra-container-image=${POD_INFRA_CONTAINER_IMAGE}
+    image: rancher/k8s:v1.5.2-rancher2
```

Comment:
Image will be v1.5.2-rancher1-1 as there is no new image for k8s, just the package changes